extensions/amp-story/1.0/amp-story.js

```diff
@@ -824,6 +825,16 @@ export class AmpStory extends AMP.BaseElement {
           accessService.areFirstAuthorizationsCompleted();
       accessService.onApplyAuthorizations(
           () => this.onAccessApplyAuthorizations_());
+
+      const firstPage = this.pages_[0].element;
+      // First amp-story-page can't be paywall protected.
+      // Throws an error during development, but just ignore the amp-access rule
+      // in production.
+      user().assert(
```

Comment:
Maybe we should also remove the attribute from the DOM?  It would be nice to provide the consistency that you can tell what's going on by looking at the DOM.

Reply:
I don't have a strong opinion on that, but since we display an error message, if a developer tries to use the devtools to look at what they should remove, and see nothing, it might be confusing.
At the same time, if we do what you suggest, we could get rid of the `bypassPaywall` statements in the `switchTo` method.
Do you have any preference?

Reply:
Wrong button!  My preference would be for removing the DOM attribute and the `bypassPaywall` special case.

Keeping the DOM consistent also makes it easier for us to debug, since the DOM is served unobfuscated but the JS is minified.